food-truck-frontend/src/pages/truck/edit/[truckId].tsx

```
@@ -97,6 +98,8 @@ class EditTruck extends Component<TruckProps, TruckComponentState> {
           message: `Failed to update truck details: ${JSON.stringify(err)}`
         })
       });
+
+    api.post(`/reindex`).catch();
```

Same issue with exceptions getting swallowed.